llvm/lib/Target/X86/X86FixupSetCC.cpp

```diff
@@ -136,6 +136,12 @@ bool X86FixupSetCCPass::runOnMachineFunction(MachineFunction &MF) {
           .addReg(ZeroReg)
           .addReg(Reg0)
           .addImm(X86::sub_8bit);
+
+      // Transfer the debug-instr-number to the setcc.
```

Comment:
Teeeeechhnnically we're not transferring the number, we're recording a substitution: if setcc already has a number, it'll retain it, and multiple variable locations will end up referring to the setcc. IMO "redirect" or "record a substitution" would be better phrasing?